src/style.css

```diff
@@ -0,0 +1,114 @@
+@font-face {
+  font-family: "Avenir", sans-serif;
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.bigImg {
+  margin-top: 25px;
+  margin-bottom: 26px;
+  margin-left: 25px;
+  height: 19px;
+  width: 19px;
+}
+
+.bigBar {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  background-color: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  display: flex;
+  align-items: center;
+  border-radius: 4px;
+  margin-top: 20px;
+  height: 70px;
+}
+
+.bigBar input {
```

Comment:
do not use tag selectors, use classes instead

Reply:
done